Dockerfiles/manifests/cluster-agent/cluster-agent.yaml

```diff
@@ -10,20 +10,25 @@ spec:
         app: datadog-cluster-agent
       name: datadog-agent
     spec:
-      serviceAccountName: datadog-cluster-agent
+      serviceAccountName: dca
       containers:
       - image: datadog/cluster-agent:latest
         imagePullPolicy: Always
         name: datadog-agent
         env:
           - name: DD_API_KEY
-            value: <YOUR_API_KEY>
+            value: '<YOUR_API_KEY>'
+          # Optionally reference an APP KEY for the External Metrics Provider.
+          #- name: DD_APP_KEY
+          #  value: '<YOUR_APP_KEY>'
           - name: DD_COLLECT_KUBERNETES_EVENTS
             value: "true"
           - name: DD_LEADER_ELECTION
             value: "true"
-         - name: DD_CLUSTER_AGENT_AUTH_TOKEN
-           value: <Thirty_2_characters_long_token>
+          - name: DD_CLUSTER_AGENT_AUTH_TOKEN
+            value: <Thirty_2_characters_long_token>
```

Comment:
If it's placeholder it should be with the same format as the other one:

`<THIRTY_2_CHARACTERS_LONG_TOKEN>`